test/test_abi_variants.py

```diff
@@ -156,6 +157,7 @@ def test():
 )
 
 
+@utils.skip_if_pyodide(reason="Doesn't work for some reason")
```

Comment:
Well Pyodide *does* support abi3, it is just pointless: you can build a `cp311-abi3-emscripten_3_1_32_wasm32` wheel but it won't work with the Python 3.10 builds of Pyodide because they used a different version of Emscripten. But the wheel is created exactly in the normal way with the limited ABI etc and it gets loaded just fine too. For instance some of the rust wheels are abi3. So I don't think abi3 is the trouble here.

Reply:
FYI, ABI3 is not all that common, but wheels that avoid the CABI are somewhat common. IMO you need to have the pyodide major version in the pyodide tag, so that `py3-none-pyodide(_22)_wasm32` will work correct - without the pyodide version, you can't tell where it can be loaded.

Reply:
Well it currently seems to just ignore the request for it to be tagged `py3-none-emscripten_wasm32` and tags it `cp311-cp311-emscripten_wasm32`